example/data.js

```diff
@@ -73,6 +73,22 @@ const bumpSyncToken = function(cal) {
   cal.syncToken = parts.slice(0, -1).join('/') + '/' + (parseInt(parts[parts.length - 1]) + 1);
 };
 
+const eventFilter = (start, end, eventStart, eventEnd) =>
+  (!end || end < eventStart) || (!start || start > eventEnd)
+
+const recurringEventFilter = (start, end, event) => {
+  if (!event.recurring) {
+    return false;
+  }
+
+  const eventStart = event.startDate
```

Comment:
Please match the coding style of the repository and include semicolons at the end of statements.

Reply:
Seems like `eslint` is not linting the example files. I can add a commit or create a new PR to solve this. What do you think?